tasks/setup-RedHat.yml

```diff
@@ -9,6 +9,25 @@
   ignore_errors: true
   when: ansible_distribution_major_version|int < 7
 
+- name: Add Varnish repository.
+  command: >
+    rpm --nosignature -i https://repo.varnish-cache.org/redhat/varnish-4.0.el7.rpm
```

Comment:
We can use the `yum` module for this instead. From that module's [examples](http://docs.ansible.com/ansible/yum_module.html):

```
- name: install the nginx rpm from a remote repo
  yum: name=http://nginx.org/packages/centos/6/noarch/RPMS/nginx-release-centos-6-0.el6.ngx.noarch.rpm state=present
```